{
  "comments": [
    {
      "key": {
        "uuid": "2293feca_b934bb3c",
        "filename": "liblog/event_tag_map.cpp",
        "patchSetId": 1
      },
      "lineNbr": 468,
      "author": {
        "id": 1004277
      },
      "writtenOn": "2017-04-05T20:19:13Z",
      "side": 1,
      "message": "Could repurpose fail_close for a common cleanup case and close anything not -1 just for peace of mind.",
      "revId": "191afba9bd17d7302a5bfb16d60ff12198336cc5",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "ac13c0ff_7deda57c",
        "filename": "liblog/event_tag_map.cpp",
        "patchSetId": 1
      },
      "lineNbr": 477,
      "author": {
        "id": 1004277
      },
      "writtenOn": "2017-04-05T20:19:13Z",
      "side": 1,
      "message": "Some of these could be -1 (if error at line 415 and which \u003e 0)?  Could skip if so.",
      "range": {
        "startLine": 477,
        "startChar": 45,
        "endLine": 477,
        "endChar": 61
      },
      "revId": "191afba9bd17d7302a5bfb16d60ff12198336cc5",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}